server.js

```diff
@@ -1,195 +1,31 @@
-          price: 26.75,
-          title: 'Ham and cheese',
-          description: 'Ham and cheese ham and cheese ham and cheese ham and cheese ham and cheese',
-          tags: [],
-        },
-        {
-          id: '0-1',
-          imgPath: '/margarita.png',
-          price: 20.5,
-          title: 'Margarita',
-          description: 'Margarita margarita margarita margarita margarita margarita margarita',
-          tags: [],
-        },
-        {
-          id: '0-2',
-          imgPath: '/pepperoni.png',
-          price: 20.5,
-          title: 'Pepperoni',
-          description: 'Pepperoni pepperoni pepperoni pepperoni pepperoni',
-          tags: [],
-        },
-        {
-          id: '0-3',
-          imgPath: '/vegetable.png',
-          price: 17.3,
-          title: 'Vegetable',
-          description: 'Vegetable vegetable vegetable vegetable vegetable',
-          tags: ['0', '1'],
-        },
-        {
-          id: '0-4',
-          imgPath: '/italian.png',
-          price: 28.0,
-          title: 'Italian',
-          description: 'Italian italian italian italian italian italian italian',
-          tags: [],
-        },
-      ],
-    },
-
-    {
-      id: '1',
-      title: 'Pasta',
-      items: [],
-    },
-
-    {
-      id: '2',
-      title: 'Sandwiches',
-      items: [],
-    },
-
-    {
-      id: '3',
-      title: 'Soup',
-      items: [],
-    },
-
-    {
-      id: '4',
-      title: 'Salads',
-      items: [],
-    },
-
-    {
-      id: '5',
-      title: 'Sides',
-      items: [],
-    },
-
-    {
-      id: '6',
-      title: 'Deserts',
-      items: [],
-    },
-
-    {
-      id: '7',
-      title: 'Drinks',
-      items: [],
-    },
-  ],
-  orders: {
-    ordered: [],
-    baking: [],
-    finishing: [],
-    served: [],
-  },
-  filters: [
-    {
-      id: '0',
-      name: 'vegetarian',
-    },
-    {
-      id: '1',
-      name: 'vegan',
-    },
-    {
-      id: '2',
-      name: 'tag0',
-    },
-    {
-      id: '3',
-      name: 'tag1',
-    },
-    {
-      id: '4',
-      name: 'tag2',
-    },
-  ],
-  discounts: [0.1],
-};
-
-app.get('/categories', (request, response) => {
-  const categories = data.categories.map((elem) => ({
-    id: elem.id,
-    title: elem.title,
-  }));
-  response.json(categories);
-});
-
-app.get('/items', (request, response) => {
-  const { id } = request.query;
-  const selectedCategory = data.categories.find((elem) => elem.id === id);
-  response.json(selectedCategory?.items);
-});
-
-app.get('/orders', (request, response) => {
-  response.json(data.orders);
-});
-
-app.get('/filters', (request, response) => {
-  response.json(data.filters);
-});
-
-app.get('/discounts', (request, response) => {
-  response.json(data.discounts);
-});
-
-app.post('/orders', (request, response) => {
-  if (!request.body) {
-    return response.sendStatus(400);
+// db-connection
+mongoose.connect(dbConn, { useUnifiedTopology: true, useNewUrlParser: true }, (err) => {
```

Comment:
Could we create `app.js` in `server` directory which contains `createApp(dbConn)` function. It should initialize express `app` instance, connect to `db` (waits for promise resolve`) and returns `app`. It will be super easier to unit tests later